modules/onboard-gcp-ms-sql-server/outputs.tf

```diff
@@ -0,0 +1,14 @@
+output "gcp-ms-sql-server-instance" {
+  description = "Google MS SQL Server instance."
```

Comment:
can we use reference the Cloud SQL service here?

Reply:
So like, `Cloud SQL for SQL Server instance` instead or am I misunderstanding what you mean?

Reply:
yeah! or `Google Cloud SQL for SQL Server instance`, up to you. It doesn't have to necessarily be here but it'd be nice to align some naming, and I do think we name the `RDS` or `Aurora` service for AWS db instances for example